Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -298,7 +298,9 @@ - (CGRect)caretRectForPosition:(UITextPosition *)position
     return CGRectZero;
   }
 
-  return [super caretRectForPosition:position];
+   CGRect originalRect = [super caretRectForPosition:position];
+   originalRect.size.height *= 0.75;
+   return CGRectMake(originalRect.origin.x, originalRect.origin.y + 4 , originalRect.size.width, originalRect.size.height);
```

Comment:
This assumes that originalRect's height < 4, so we can offset it by this much, which is not safe.